tensorflow/tools/pip_package/setup.py

```diff
@@ -158,6 +158,22 @@ def standard_or_nightly(standard, nightly):
       _VERSION + ';platform_system=="Windows"',
   ]
 
+EXTRA_PACKAGES = {}
+EXTRA_PACKAGES['with-cuda'] = [
```

Comment:
Thanks! This structure makes sense because putting logic inside of a dict constructor is usually awkward.
After thinking about it some more I think `tensorflow[and-cuda]` is my preferred modifier; can you change that and then convert this to a regular PR?

Reply:
Done.